lib/fix.js

```diff
@@ -1,8 +1,7 @@
-/** @babel */
```

Comment:
Turns out, the `/** @babel */` comments are causing atom to transpile files with babel

I wasn't able to find much documentation about it.. but it appears atom uses babel@6, which isn't capable of parsing option catch blocks `} catch {`

It looks like babel was just transpiling es modules to require statements?

I tried removing them to see if CI passes. I imagine the transpile step would add some time to starting up atom? Looking at timecop, `atom-xo-lint` takes about 90ms to startup on my machine

Reply:
I'd be interested to see if keeping the es modules, and somehow using typescript would help at all? I noticed some JSDoc style comments, and talk in previous PRs about possibly adding some type checking :eyes: